apps/explorer/src/app/components/table/index.tsx

```diff
@@ -25,6 +25,21 @@ interface TableCellProps extends ThHTMLAttributes<HTMLTableCellElement> {
 }
 
 export const Table = ({ children, className, ...props }: TableProps) => {
+  const classes = classnames(className, 'overflow-x-auto whitespace-nowrap');
```

Comment:
chore (non-blocking): I think we need to add our key/value table to UI toolkit from TFE as this covers most basic tables and would probably reduce what we needed to keep here in this project

Reply:
Ah ok cool, I'll do that when I move on to governance shortly

Reply:
#171